### DISCUSSION TITLE
Syncing registration using n8n

### DISCUSSION THREAD

**Discussion by tobsowo** [alfio-event/alf.io#1277] [Q&A]
Please, can someone help me with how I can use n8n to sync registration details with Mautic? Mautic already works with n8n.

Thank you.

**Comment by cbellone** [comment]
Hi, 
this can be done using our [Extension engine](https://alf.io/docs/reference/extensions/)

From your description I believe that the extension you need is the [TICKET_ASSIGNED](https://alf.io/docs/reference/extensions/reference/ticket/#ticket-assigned) which provides you with:

- ticket basic data (First name, last name, email)
- additional data you defined in the event configuration

hope this helps

**Reply by tobsowo** [reply]
Thank you.
I'll check it out.
